### DISCUSSION TITLE
ImportError:

### DISCUSSION THREAD

**Discussion by badouzeum** [brentvollebregt/auto-py-to-exe#416] [Q&A] [answered]
je veux Convertir un fichier python .py en éxecutable .exe avec le module python auto py to exe mais essai d'ouvrir la librairie "auto py to exe " je reçois le message suivant:
 
 File "C:\Users\PRECISION\AppData\Roaming\Python\Python311\site-packages\greenlet\__init__.py", line 29, in <module>
    from ._greenlet import _C_API # pylint:disable=no-name-in-module
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ImportError: DLL load failed while importing _greenlet: Le module spécifié est introuvable.

**Comment by brentvollebregt** [comment, marked as answer]
Copy of #415 (which is a duplicate of #325) - take a look at [this solution](https://github.com/brentvollebregt/auto-py-to-exe/issues/325#issuecomment-1313939303).
